packages/content/src/relationships/ArbitraryRelationshipCreationContent.ts

```diff
@@ -0,0 +1,26 @@
+import { ISerializable, Serializable, serialize, type, validate } from "@js-soft/ts-serval";
+import { ContentJSON } from "../ContentJSON";
+
+export interface ArbitraryRelationshipCreationContentJSON extends ContentJSON {
+    "@type": "ArbitraryRelationshipCreationContent";
+    value: any;
+}
+
+export interface IArbitraryRelationshipCreationContent extends ISerializable {
+    value: any;
+}
+
+@type("ArbitraryRelationshipCreationContent")
+export class ArbitraryRelationshipCreationContent extends Serializable implements IArbitraryRelationshipCreationContent {
+    @serialize()
+    @validate()
+    public value: any;
```

Comment:
@stnmtz @tnotheis wouldn't `unknown` be the better option here (also in the interface and JSON)?

Reply:
Isn't `unknown` **always** the better choice? At least I can't see any major downside in this case.